classes/ConfigurationTest.php

```diff
@@ -158,7 +158,7 @@ public static function test_apache_mod_rewrite()
 
     public static function test_new_phpversion()
     {
-        return version_compare(PHP_VERSION, '7.1.3', '>=');
+        return static::test_phpversion();
```

Comment:
We can remove this function, edit the `getDefaultTestsOp` and use `'phpversion'` instead of `'new_phpversion`'?

Reply:
Technically, it'd be a breaking change. I'll edit the consumer and deprecate the function though.

Reply:
Technically, it'd be a breaking change. I'll edit the consumer and deprecate the function though.

Reply:
Found the original intent for this test. It was for future-proofing: https://github.com/PrestaShop/PrestaShop/commit/b7753c1b547e3a455a6c691b8d90da1fc8639a07#diff-04f3684340e9328d0376ebec86a87523

Maybe it would be interesting to use it to push for PHP 7.2?

Reply:
But methods name must change.
It's not about the new php version, but the recommended one :thinking:

Reply:
How about we do that in a follow-up PR?